config.make

```diff
@@ -128,18 +128,38 @@ ZFP_INC = $(ZFP_HOME)/inc
 else
 ZFP_INC = $(ZFP_HOME)/include
 endif
+ifeq ($(wildcard $(ZFP_HOME)/lib),)
+ZFP_LIB = $(ZFP_HOME)/lib64
+else
 ZFP_LIB = $(ZFP_HOME)/lib
+endif
+
+# Check if specified individually the HDF5 include directory,
+# library directory and bin directory separated by commas, i.e. HDF5_HOME=INC,LIB,BIN
```

Comment:
If a package installer is used for hdf5, then one can't really set HDF5_HOME 
For github actions don't compile hdf5 from source and instead use the package manager, much faster :)

Reply:
> If a package installer is used for hdf5, then one can't really set HDF5_HOME

Not sure I understand. You mean that a package installer puts hdf5 in a *standard* place and so doesn't require `HDF5_HOME` to be set but the make logic here expects it set anyways? If so, maybe we just need the logic to allow for an empty `HDF5_HOME`?

BTW...there is a CMake interface now too, which I think is now preferred. Not suggesting you take that on here. Just wanted to mention it for future reference.